dashboard/15-final/app/ui/invoices/create-form.tsx

```diff
@@ -14,122 +24,163 @@ export default function Form({
   const [state, dispatch] = useFormState(createInvoice, initialState);
 
   return (
-    <div className="mx-auto max-w-sm rounded-lg border px-6 py-8 shadow-sm">
-      <h2 className="mb-6 text-xl font-semibold">Update Invoice</h2>
+    <div>
+      <nav aria-label="Breadcrumb" className="mb-6 block">
+        <ol className={clsx(lusitana.className, 'flex text-2xl')}>
+          <li aria-current="false" className="text-gray-500">
+            <Link href="/dashboard/invoices">Invoices</Link>
+            <span className="mx-3 inline-block">/</span>
+          </li>
+          <li aria-current="true">
+            <Link href="/invoices/create">Create</Link>
+          </li>
+        </ol>
+      </nav>
```

Comment:
Can we extract the breadcrumbs into separate component? Want to reduce the amount of code as the user will be asked to edit the forms.